### PR TITLE
HoC 2024 - Add OpenGraph image and tags

### PR DESCRIPTION
**Pull request by kelbyhawn**
Updates the OpenGraph image and tags for the following pages for Hour of Code 2024.
- https://hourofcode.com homepage
- https://hourofcode.com/learn 
- https://code.org/hourofcode

I also removed the code.org hoc-overview OG stuff since that page no longer exists and fixed the spacing on the nav and hamburger spacing.

## Links
Jira ticket: [ACQ-2370](https://codedotorg.atlassian.net/browse/ACQ-2370)

## Testing story
Tested locally

### CODE REVIEW

File: pegasus/sites.v3/hourofcode.com/styles/060-hoc-specific.css [hunk 1]
```diff
@@ -35,7 +35,7 @@
 }
 #hoc-nav ul {
   list-style: none;
-  margin: .2em 0;
+  margin: 1px 0;
```
Comment (kelbyhawn) [inline pegasus/sites.v3/hourofcode.com/styles/060-hoc-specific.css:38]: ### Before
nav_before

### After
nav_after

File: pegasus/sites.v3/hourofcode.com/styles/060-hoc-specific.css [hunk 2]
```diff
@@ -178,6 +178,7 @@ iframe {
   }
   #hoc-nav-hamburger {
     display: inline-block;
+    margin-top: -8px;
```
Comment (kelbyhawn) [inline pegasus/sites.v3/hourofcode.com/styles/060-hoc-specific.css:181]: ### Before
hamburger_before

### After
hamburger_after

File: pegasus/src/social_metadata.rb [hunk 3]
```diff
@@ -81,9 +81,9 @@ def get_social_metadata_for_page(request)
     },
     "hourofcode.com" => {
       "default" => {
-        title: hoc_s(:hoc2023_social_creativity_with_ai_title),
-        description: hoc_s(:hoc2023_social_creativity_with_ai_desc),
-        image: images[:hoc_2023_social]
+        title: hoc_s("hoc_2024.social.title"),
+        description: hoc_s("hoc_2024.social.desc"),
+        image: images[:hoc_2024_social]
```
Comment (kelbyhawn) [inline pegasus/src/social_metadata.rb:86]: hoc_homepage

File: pegasus/src/social_metadata.rb [hunk 4]
```diff
@@ -134,23 +134,16 @@ def get_social_metadata_for_page(request)
     },
     "learn" => {
       "default" => {
-        title: hoc_s(:hoc2023_social_creativity_with_ai_title),
-        description: hoc_s(:hoc2023_social_creativity_with_ai_desc),
-        image: images[:hoc_2023_social]
+        title: hoc_s("hoc_2024.social.title"),
+        description: hoc_s("hoc_2024.social.desc"),
+        image: images[:hoc_2024_social]
```
Comment (kelbyhawn) [inline pegasus/src/social_metadata.rb:139]: hoc_learn

File: pegasus/src/social_metadata.rb [hunk 5]
```diff
@@ -134,23 +134,16 @@ def get_social_metadata_for_page(request)
...
-      }
-    },
-    "learn-cdo" => {
-      "default" => {
-        title: hoc_s(:social_hoc_anybody),
-        description: hoc_s(:social_hoc2022_explore_play_create),
-        image: images[:hoc_2022_social]
+        title: hoc_s("hoc_2024.social.title"),
+        description: hoc_s("hoc_2024.social.desc"),
+        image: images[:hoc_2024_social]
```
Comment (kelbyhawn) [inline pegasus/src/social_metadata.rb:146]: overview

**Review by TurnerRiley — APPROVED** [review]

**Pull request merged**